packages/fluentui/react-northstar/src/components/Tree/hooks/useStableProps.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+export function useStableProps<P>(props: P) {
+  const stableProps = React.useRef<P>(props);
+
+  React.useEffect(() => {
+    stableProps.current = props;
+  });
```

Comment:
This effect will be run on every single rerender, is that intended ?

Reply:
Yes. I would like to replace this with another hook on next iteration

Reply:
Per or offline discussion, I see why this is necessary now